### PR TITLE
Readme update representing the final state of the project

### PR DESCRIPTION
**Pull request by GuusVink**
All that needs to be said from our side (if I did not forget anything)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -75,13 +71,7 @@ A list of Java language features which are included in the analyzer can be found
 
 ## Support Libraries
 
-There is currently no support library available. 
-
-
-## Sample applications renderers
-This repository contains two sample applications that make use of the Java Spring framework ([Spring events](/ldj-spring-event-example) & [Spring AMQP](/ldj-spring-amqp-example)).
-For both these sample applications a [renderer](/ldj-spring-renderer-example) has been made (which generates a UML sequence diagram). This may serve as a source of inspiration for creating your own renderers.
-
```
Comment (wandernauta) [inline README.md:84]: This seems like it is also still true and could be kept.

I would perhaps spell the header 'Sample applications **and** renderers'.

**Review by wandernauta — COMMENTED** [review]
Thanks! I've put a few comments where I had questions about the suggested changes.

**Comment by GuusVink** [comment]
Thanks for catching these unintended removals.
It seems previous changes to the readme were not made in the template but in the readme directly. When rerendering the Readme, these older sections were (unintentionally) removed.
I will fix these when I have some time

**Comment by GuusVink** [comment]
New version with changes to address the issues

**Review by wandernauta — APPROVED** [review]
Looks good to me!

**Pull request merged**